root/dashboard/swag-f2b-host.py

```diff
@@ -0,0 +1,32 @@
+import json
```

Comment:
There's a lot of duplicate code both here and in the php, better change the current code to handle the host f2b